osu.Game.Rulesets.Catch/UI/Catcher.cs

```diff
@@ -21,6 +21,8 @@ namespace osu.Game.Rulesets.Catch.UI
 {
     public class Catcher : Container, IKeyBindingHandler<CatchAction>
     {
+        public static Color4 DefaultHyperDashColour { get; } = Color4.Red;
```

Comment:
Make all `static Color4`'s in this PR into `static readonly Color4`'s.

Reply:
that'll rename `DefaultHyperDashColour` to `DEFAULT_HYPER_DASH_COLOUR` is that fine? because that's why I used a static getter-only property just like how `SkinInfo` provides `Default`.

Reply:
That's what the convention is, so that's what it will be.